src/data/modifiers.txt

```diff
@@ -3676,47 +3711,47 @@ Item	stuffed carpenter	Synergetic
-# teflon swim fins: Makes you a better diver
 Item	teflon swim fins	Moxie Percent: -5, Initiative: +25, Single Equip, Initiative Penalty: [10*env(underwater)], Item Drop Penalty: [10*env(underwater)], Meat Drop Penalty: [10*env(underwater)]
-# terra cotta tie tack: Lets you unleash a terra cotta army on your foes
+# teflon swim fins: Makes you a better diver
 Item	terra cotta tie tack	Experience (Moxie): +3, Meat Drop: +20, Single Equip
-# terra cotta truss: Lets you unleash a terra cotta army on your foes
+# terra cotta tie tack: Lets you unleash a terra cotta army on your foes
 Item	terra cotta truss	Experience (Muscle): +3, Initiative: +40, Single Equip
+# terra cotta truss: Lets you unleash a terra cotta army on your foes
 Item	the Ley Incursion's waist	Mysticality: +15, Mysticality Percent: +25, Experience (Mysticality): +5, Single Equip
 Item	The Lost Glasses	Reduce Enemy Defense: 15, Hot Resistance: +2, Single Equip
 Item	The One Mood Ring	Muscle Percent: [5*G], Mysticality Percent: [5*G], Moxie Percent: [5*G], Monster Level: [25-5*G], Single Equip
 Item	The Ring	Muscle: +3, Mysticality: +3, Moxie: +3, Spooky Damage: +40, Single Equip
-# The Superconductor's CPU: (based on Crimbo 22 Elf Gratitude)
 Item	The Superconductor's CPU	Maximum HP: [floor(sqrt(pref(elfGratitude)))], Maximum MP: [floor(sqrt(pref(elfGratitude)))], Single Equip
+# The Superconductor's CPU: (based on Crimbo 22 Elf Gratitude)
 Item	thicksilver ring	Damage Absorption: +30, Damage Reduction: 60, Single Equip
 Item	third ear	Initiative: +50, Single Equip
 Item	ticksilver ring	Adventures: +6, PvP Fights: +4, Single Equip
 Item	tie-dyed fannypack	Spooky Damage: +2, Stench Damage: +2, Hot Damage: +2, Cold Damage: +2, Sleaze Damage: +2, Item Drop: +10, Single Equip
-# Time Bandit Badge of Courage: Proves your worth to the Time Bandits
 Item	Time Bandit Badge of Courage	Muscle Limit: 50, Mysticality Limit: 50, Moxie Limit: 50
+# Time Bandit Badge of Courage: Proves your worth to the Time Bandits
 Item	time halo	Adventures: +5, Unarmed, Single Equip
-# Time Lord Badge of Honor: Proves your worth to the Time Lords
 Item	Time Lord Badge of Honor	Muscle Limit: 50, Mysticality Limit: 50, Moxie Limit: 50
-# time-twitching toolbelt: +60% Item Drops in the Time-Twitching Tower
-# time-twitching toolbelt: Reveals various Time-Forgotten Secrets
-# time-twitching toolbelt: (also in the Time-Twitching Tower)
+# Time Lord Badge of Honor: Proves your worth to the Time Lords
 # time-twitching toolbelt	Muscle: +15, Mysticality: +15, Moxie: +15, Single Equip, Item Drop: [60*zone(Twitch)], Free Pull: [pref(timeTowerAvailable,true)]
 Item	time-twitching toolbelt	Muscle: +15, Mysticality: +15, Moxie: +15, Single Equip, Item Drop: [60*zone(Twitch)], Variable
+# time-twitching toolbelt: (also in the Time-Twitching Tower)
+# time-twitching toolbelt: +60% Item Drops in the Time-Twitching Tower
+# time-twitching toolbelt: Reveals various Time-Forgotten Secrets
```

Comment:
this is another block where alphabetizing the existing comments leads to a strange read.

Reply:
Hmm

Reply:
a naive rule like "brackets always go to the end" makes it moderately readable